randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -946,15 +946,23 @@ Extra - asset_id: collision_camera_012
   * Extra - left_shield_def: actordef:actors/props/db_hdoor_b1_001/charclasses/db_hdoor_b1_001.bmsad
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
-  > Ledge before Slide Jump
+  > Door to Transport to Artaria
       All of the following:
           Morph Ball
           Any of the following:
               Varia Suit
-              Heat/Cold Runs (Intermediate) and Heat Damage ≥ 400
+              Heat/Cold Runs (Intermediate) and Heat Damage ≥ 450
           Any of the following:
               Use Spin Boost
-              Slide and Slide Jump (Intermediate)
+              All of the following:
+                  Slide
+                  Any of the following:
+                      Slide Jump (Intermediate)
+                      All of the following:
+                          Slide Jump (Beginner)
+                          Any of the following:
+                              Flash Shift
+                              Cross Bomb and Movement (Beginner)
```

Comment:
If you're removing the ledge node (which I do admit is unnecessary), then Intermediate Slide Jump is the only thing that works without spin boost. Cross Bomb and Flash won't help you get into the tunnel above the lava. Disregarding that the second jump is actually more lenient due to being able to save yourself if you fail and is why it was rated lower, you need to do two slide jumps for the whole sequence and failing either will plummet you into the lava.

Reply:
I'm going to insist that the second one should be intermediate even though it can be retried by jumping back from the slope. The timing is relatively tight, and you have to jump surprisingly late. I'm convinced that the first one has more leniency, and is fine as beginner. Though, it's possible to make a case for adding a beginner wall jump. It's not necessarily an important point, but there's a solid number of options for the second jump that are just completely irrelevant if the first jump is not rated lower than the second one. I'm also convinced that other people agree with me on this rating.

Reply:
Based on discussion in Discord, perhaps we make it intermediate and advanced, then call it a day.

Reply:
Done.